daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy-without-cli.md

```diff
@@ -0,0 +1,299 @@
+---
+type: docs
+title: "Deploy Dapr at a Kubernetes cluster"
+linkTitle: "Deploy Dapr without CLI"
+weight: 21000
+description: "Follow these steps to deploy Dapr at Kubernetes without cluster permissions."
+aliases:
+    - /getting-started/install-dapr-kubernetes/
+---
+
+This part describes the process of installation Dapr without CLI or Helm when you have no cluster permissions. Such situation may arise when you have a project in OKD or namespace in Kubernetes. In this case you have no cluster permission and can't deploy the operator. You can't also address cluster admin to install Dapr operator because you are the only one who needs it.
```

Comment:
```suggestion
This page describes the process of installation Dapr without CLI or Helm when you have no cluster permissions. Such situation may arise when you have a project in OKD or a namespace in Kubernetes. In this case you have no cluster permission and can't deploy the operator. You can't also address cluster admin to install Dapr operator because you are the only one who needs it.
```

Reply:
Just to be clear, this example only deploys a local development environment to the kubernetes cluster, so this should be  explicitly mentioned in the title. This is not about setting up a production environment which should use Helm or the CLI. This is not clear in the overview and it would be good to add.

Reply:
I agree with you about example. But you could deploy three instances of placement and dashboard. It will be similar to production environment